msprime/mutations.py

```diff
@@ -91,15 +132,21 @@ def __str__(self):
         return s
 
 
-class SlimMutationModel(_msprime.SlimMutationModel):
-    asdict = safe_asdict
+class SlimMutationModel(
+    _msprime.SlimMutationModel, MutationModel, metaclass=core.consistent_metaclass()
+):
+    pass
```

Comment:
Mind throwing this in there?
```
def __str__(self):
    s = f"Mutation model for SLiM mutations of type m{self.type}\n"
    s += f"  next ID: {self.next_id}"
    return s
```
(note: the `m` in front of `type` is not a typo)

Reply:
https://github.com/tskit-dev/msprime/pull/1176/commits/7dbf8a07a59a9c5b68b35afd6389a99233f61712